pages/docs/form/slider.mdx

```diff
@@ -111,6 +114,89 @@ accomplished using the `step` prop.
 </Slider>
 ```
 
+### Slider with custom labels and marks
+
+You can have custom labels and marks by using `SliderMark` component
+
+```jsx
+function SliderMarkExample() {
+  const [sliderValue, setSliderValue] = useState(50)
+  return (
+    <Slider aria-label='slider-ex-6' onChange={(val) => setSliderValue(val)}>
+      <SliderMark value={25} mt='1' ml='-2.5' fontSize='sm'>
+        25%
+      </SliderMark>
+      <SliderMark value={50} mt='1' ml='-2.5' fontSize='sm'>
+        50%
+      </SliderMark>
+      <SliderMark value={75} mt='1' ml='-2.5' fontSize='sm'>
+        75%
+      </SliderMark>
+      <SliderMark
+        value={sliderValue}
+        textAlign='center'
+        bg='blue.500'
+        color='white'
+        mt='-10'
+        ml='-5'
+        w='12'
+      >
+        {sliderValue}%
+      </SliderMark>
+      <SliderTrack>
+        <SliderFilledTrack />
+      </SliderTrack>
+      <SliderThumb />
+    </Slider>
+  )
+}
+```
+
+And you can also use `Tooltip` with `SliderThumb`
+
+```jsx
+function SliderThumbWithTooltip() {
+  const [sliderValue, setSliderValue] = React.useState(5)
+  const [showTooltip, setShowTooltip] = React.useState(false)
+  return (
+    <Slider
+      id='slider'
+      defaultValue={5}
+      min={0}
+      max={100}
+      colorScheme='teal'
+      onChange={(v) => setSliderValue(v)}
+      onMouseEnter={() => setShowTooltip(true)}
+      onMouseLeave={() => setShowTooltip(false)}
+    >
+      <SliderMark value={25} mt='1' ml='-2.5' fontSize='sm'>
+        25%
+      </SliderMark>
+      <SliderMark value={50} mt='1' ml='-2.5' fontSize='sm'>
+        50%
+      </SliderMark>
+      <SliderMark value={75} mt='1' ml='-2.5' fontSize='sm'>
+        75%
+      </SliderMark>
+      <SliderTrack>
+        <SliderFilledTrack />
+      </SliderTrack>
+      <Tooltip
+        hasArrow
+        bg='teal.500'
+        color='white'
+        placement='top'
+        closeOnClick={false}
```

Comment:
Actually this prop is redundant as well. Otherwise it's ready to be merged!

Reply:
Done!